.tool-versions

```diff
@@ -1,2 +1,2 @@
-erlang 20.2.2
+erlang 20.2
```

Comment:
Why?

Reply:
I was having difficulties installing the right patch version number so I gave up.